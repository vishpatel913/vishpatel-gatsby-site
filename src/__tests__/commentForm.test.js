import React from "react";
import renderWithTheme from "./utils/test-renderer";
import CommentForm from "../components/commentForm";

describe("CommentForm", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme(<CommentForm slug="test" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
