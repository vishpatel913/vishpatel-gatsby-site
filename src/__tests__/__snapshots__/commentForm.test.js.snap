// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommentForm renders correctly 1`] = `
.c4 {
  display: inherit;
  font-size: 18px;
  margin-right: 4px;
  vertical-align: text-top;
}

.c0 {
  background: white;
  border: solid 1px #E0E0E0;
  border-radius: 4px;
  padding: 0.5rem;
  margin-right: 1rem;
  margin-bottom: 1rem;
  width: 40%;
  box-sizing: border-box;
}

.c0::-webkit-input-placeholder {
  font-weight: lighter;
}

.c0::-moz-placeholder {
  font-weight: lighter;
}

.c0:-ms-input-placeholder {
  font-weight: lighter;
}

.c0::placeholder {
  font-weight: lighter;
}

.c1 {
  display: block;
  width: 100%;
  background: white;
  border: solid 1px #E0E0E0;
  border-radius: 4px;
  padding: 0.5rem;
  margin-right: 1rem;
  margin-bottom: 1rem;
}

.c1::-webkit-input-placeholder {
  font-weight: lighter;
}

.c1::-moz-placeholder {
  font-weight: lighter;
}

.c1:-ms-input-placeholder {
  font-weight: lighter;
}

.c1::placeholder {
  font-weight: lighter;
}

.c2 {
  cursor: pointer;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-evenly;
  -webkit-justify-content: space-evenly;
  -ms-flex-pack: space-evenly;
  justify-content: space-evenly;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  font-size: 12px;
  max-width: 25em;
  color: white;
  background: #35cdbc;
  text-align: center;
  line-height: 0;
  padding: 1em;
  margin: auto;
  -webkit-transition: all 0.3s ease 0s;
  transition: all 0.3s ease 0s;
  border-radius: 4px;
  opacity: 0;
}

@media (max-width:768px) {
  .c0 {
    width: 100%;
    margin-right: 0;
  }
}

@media (max-width:768px) {
  .c2 {
    margin-left: auto;
  }
}

<div
  className=""
>
  <h2>
    Leave a Comment
  </h2>
  <form
    data-netlify="true"
    data-netlify-honeypot="bot-field"
    disabled={true}
    id="comment-form"
    method="POST"
    name="test-comments"
    onSubmit={[Function]}
  >
    <input
      name="form-name"
      type="hidden"
      value="test-comments"
    />
    <input
      name="bot-field"
      type="hidden"
    />
    <input
      name="slug"
      type="hidden"
      value="test"
    />
    <input
      aria-label="Name"
      className="c0"
      name="name"
      placeholder="Name*"
      required={true}
      type="text"
    />
    <input
      aria-label="Email"
      className="c0"
      name="email"
      placeholder="Email"
      type="email"
    />
    <textarea
      aria-label="Message"
      className="c1"
      name="message"
      placeholder="Message*"
      required={true}
      rows="5"
    />
    <button
      className="c2"
      type="submit"
    >
      Submit
    </button>
  </form>
  <div
    className="c3"
    id="success-toast"
  >
    <span
      className="c4"
    >
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 512 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M504 256c0 136.967-111.033 248-248 248S8 392.967 8 256 119.033 8 256 8s248 111.033 248 248zM227.314 387.314l184-184c6.248-6.248 6.248-16.379 0-22.627l-22.627-22.627c-6.248-6.249-16.379-6.249-22.628 0L216 308.118l-70.059-70.059c-6.248-6.248-16.379-6.248-22.628 0l-22.627 22.627c-6.248 6.248-6.248 16.379 0 22.627l104 104c6.249 6.249 16.379 6.249 22.628.001z"
        />
      </svg>
    </span>
    Comment submitted and pending review
  </div>
</div>
`;
